pytest.ini

```
@@ -0,0 +1,5 @@
+[pytest]
+norecursedirs = .eggs build *.egg-info
+python_files = tests/*.py
+addopts = --assert=plain -n auto --dist=loadfile --maxprocesses=8 --disable-warnings --tb=short
```

```suggestion
addopts = --assert=plain -n auto --dist=load --maxprocesses=8 --disable-warnings --tb=short
```

dist=load parallelizes better and these tests have no issues being parallelized